Objects/memoryobject.c

```diff
@@ -1589,7 +1600,7 @@ memory_getbuf(PyObject *_self, Py_buffer *view, int flags)
 
 
     view->obj = Py_NewRef(self);
-    self->exports++;
+    _Py_atomic_add_ssize(&self->exports, 1);
```

Comment:
Wrap this atomic and the one below in an `#ifdef Py_GIL_DISABLED`. We don't need atomics if the GIL is enabled.

Reply:
Hi @ZeroIntensity ,

Got it! 😊

I have added a macro `Py_GIL_DISABLED` and keeping original code when non-gil is not enabled.

Best Regards,
Edward